x-pack/legacy/plugins/siem/server/plugin.ts

```diff
@@ -151,12 +153,24 @@ export class Plugin {
     });
 
     if (plugins.alerting != null) {
-      const type = signalRulesAlertType({
+      const { host, port, protocol } = core.http.getServerInfo();
+      const kibanaUrl = `${protocol}://${host}:${port}`;
```

Comment:
Not sure if this will work on cloud -- may want to touch base with the platform team to ensure this will be fine there.